node/network/protocol/src/peer_set.rs

```diff
@@ -118,7 +111,7 @@ impl PeerSet {
 	/// of the main protocol name reported by [`PeerSetProtocolNames::get_main_name()`].
 	pub fn get_main_version(self) -> ProtocolVersion {
 		match self {
-			PeerSet::Validation => ValidationVersion::V1.into(),
+			PeerSet::Validation => ValidationVersion::VStaging.into(),
```

Comment:
this doesn't look like it should be merged as-is. the `main` version should always be a real version, not a staging version. In the async backing branch, we've gated this on a `feature = "network-protocol-staging"`.

Reply:
I will add the same, maybe we can eventually merge my PR and have a single network protocol upgrade.

Reply:
done